File: PHP/Aksesoris.php
<?php
require_once "PetShop.php";

class Aksesoris extends PetShop {
    protected $jenis;
    protected $bahan;
    protected $warna;

    public function __construct($id_produk, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna) {
        parent::__construct($id_produk, $nama_produk, $harga_produk, $stok_produk, $foto_produk);
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    public function getInfo() {
        return array_merge(parent::getInfo(), [$this->jenis, $this->bahan, $this->warna]);
    }
}
?>

File: Java/Aksesoris.java
public class Aksesoris extends PetShop {
    protected String jenis;
    protected String bahan;
    protected String warna;

    public Aksesoris(int id_produk, String nama_produk, int harga_produk, int stok_produk, String jenis, String bahan, String warna) {
        super(id_produk, nama_produk, harga_produk, stok_produk);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    @Override
    public String tampilkanInfo() {
        return super.tampilkanInfo() + "\t" + jenis + "\t" + bahan + "\t" + warna;
    }
}
